From 73594ea04ef629c599eaddaeedbce3afded60e01 Mon Sep 17 00:00:00 2001
Message-Id: <73594ea04ef629c599eaddaeedbce3afded60e01@dist-git>
From: "Daniel P. Berrange" <berrange@redhat.com>
Date: Fri, 8 Jul 2016 09:57:58 +0200
Subject: [PATCH] virtlogd: increase max file size to 2 MB

https://bugzilla.redhat.com/show_bug.cgi?id=1351209

People debugging guest OS boot processes and reported that
the default 128 KB size is too small to capture an entire
boot up sequence. Increase the default size to 2 MB which
should allow capturing a full boot up even with verbose
debugging.

Signed-off-by: Daniel P. Berrange <berrange@redhat.com>
(cherry picked from commit 8f390596beeba94b691687a4fe3697ebf08858c7)
---
 src/logging/log_daemon_config.c | 2 +-
 src/logging/virtlogd.conf       | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/logging/log_daemon_config.c b/src/logging/log_daemon_config.c
index b584970..7bf10c6 100644
--- a/src/logging/log_daemon_config.c
+++ b/src/logging/log_daemon_config.c
@@ -128,7 +128,7 @@ virLogDaemonConfigNew(bool privileged ATTRIBUTE_UNUSED)
         return NULL;
 
     data->max_clients = 1024;
-    data->max_size = 128 * 1024;
+    data->max_size = 1024 * 1024 * 2;
     data->max_backups = 3;
 
     return data;
diff --git a/src/logging/virtlogd.conf b/src/logging/virtlogd.conf
index fbd92ff..7ef1ac3 100644
--- a/src/logging/virtlogd.conf
+++ b/src/logging/virtlogd.conf
@@ -59,8 +59,8 @@
 #max_clients = 1024
 
 
-# Maximum file size before rolling over. Defaults to 128 KB
-#max_size = 131072
+# Maximum file size before rolling over. Defaults to 2 MB
+#max_size = 2097152
 
 # Maximum number of backup files to keep. Defaults to 3,
 # not including the primary active file
-- 
2.9.0

